src/webui/webui.c

```diff
@@ -1597,6 +1598,304 @@ page_play_auth(http_connection_t *hc, const char *remain, void *opaque)
   return page_play_(hc, remain, opaque, URLAUTH_CODE);
 }
 
+
+
+#if ENABLE_HDHOMERUN_SERVER
+/**
+ * Dummy password verify callback for HDHomeRun.  We need this to
+ * ensure we can get the aa_auth information for the user in to the
+ * access_t since it is not populated by an access_get_by_username.
+ * Since the user is configured solely on the server, the "passwd" is
+ * always valid.
+ *
+ * We have the username configured on the server (not the client)
+ * since HDHomeRun clients usually do not allow entry of credentials.
+ *
+ * This is called via the "access_get" call which does IP checking
+ * (configured via the Users/IP Blocking Records tab in expert
+ * setting).
+ */
+static int
+hdhomerun_server_verify_callback(void *aux, const char *passwd)
+{
+  return 1;
+}
+
+
+static const char *hdhomerun_get_server_name(void)
```

Comment:
I'm not much of a "law ninja", but given that HDHomeRun is a registered trademark, couldn't it bring legal problems for TVHeadend to have the name "hdhomerun" in the source code? Perhaps an alias like "hdhr" could be a better fit for the functions and variables. Perhaps somebody with more insight into laws could give some more info on this..